src/pages/ExamsPage/messages.js

```diff
@@ -343,6 +343,38 @@ const messages = defineMessages({
     description: 'Description for the allowance value column in allowance table',
   },
 
+  // Allowance form
```

Comment:
Note: updated to include messages that are universal to the allowance modal form.